### ISSUE TITLE
Future audio only option? Maybe?

### ISSUE THREAD

**Issue by ToatsNotMyUsername** [tharunbirla/FetchIt#9]
Was just curious if there was ever potentially going to be an option to download just the audio from videos? Primarily YouTube in my case. Videos take up so much more space when all I need is the audio. Other platforms would be cool but, again, my main thing Is YouTube audio(s).

If not likely then is there any way for me to make the modification to the app myself? Like a particular line in the source code I can edit in the ability or something? Orrr not really?

**Comment by tharunbirla** [comment]
Hi @ToatsNotMyUsername,

Thank you for your suggestion! I’m planning to add an audio-only download option in a future release.

The app currently uses cobalt.tools for downloading YouTube videos and other platforms, except for Facebook, Twitter, and Instagram.

I’ll have to look into a workaround for this feature.

**Comment by ToatsNotMyUsername** [comment]
Awesome! I have one more suggestion/question. This one being a potentially big undertaking/challenge for some distant future you if you haven't already considered it or started on it. But does batch downloading sound feasible with the resources you use? Because essentially I've really been looking for a more optimized version of TubeMate because I like listening to music but I hate buffering in the middle of a good song and I don't have the money to pay for every single song of hundreds that I want to listen to. Tubemate is really the only thing out there that I can use on my phone. It just is very inefficient and slow and there's not much competition to make improvements a big deal to the creators.

**Comment by peterphamous** [comment]
Have you tried YTDLnis?

**Comment by ToatsNotMyUsername** [comment]
Nope, thanks for the recommendation. I'll have to check it out. I don't know if that size of project is your end goal, but good luck on any improvements you make in the future! So far it's pretty nice and quick.

**Comment by tharunbirla** [comment]
Hi @ToatsNotMyUsername,

Thanks for your patience! I’m glad to hear you enjoy the app and are interested in batch downloading.

Could you provide more information about the batch download feature? How do you envision the functionality of the batch download feature?

In the meantime, I recommend checking out [Seal](https://github.com/JunkFood02/Seal) or [YTDLnis](https://github.com/deniscerri/ytdlnis) as @peterphamous suggested. Both offer options for downloading audio and video content.

I appreciate your suggestion and will certainly consider it!

**Comment by leminsc8er** [comment]
Souce and Codes from jdownloader.org and NewPipe van be help.

**Comment by tharunbirla** [comment]
@leminsc8er need to check that out

**Comment by rr-tid** [comment]
Hi, new to this project, but the "old" DVD app doesn't support this version of Android, yours does 👍. The problem as I'm using Fetch 1.4 is that the downloaded file has a size of 0 bits. Sorry if this is a duplicate or is in the wrong location.
